mesonpy/_introspection.py

```diff
@@ -20,8 +20,8 @@ def debian_python() -> bool:
         import distutils
         try:
             import distutils.command.install
-        except ModuleNotFoundError:
-            raise ModuleNotFoundError('Unable to import distutils, please install python3-distutils')
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError('Unable to import distutils, please install python3-distutils') from exc
```

Comment:
I don't get the purpose of this change. AFAIK, within the handling of an exception, the `raise ... from` form is exactly the same as the one omitting the `from`. If anything, here I would use `... from None`.

Reply:
From https://github.com/PyCQA/flake8-bugbear#opinionated-warnings

> **B904**: Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling. See [the exception chaining tutorial](https://docs.python.org/3/tutorial/errors.html#exception-chaining) for details.

This is actually very helpful.

Reply:
TLDR: It changed the message from

```
...
During handling of the above exception, another exception occurred:
...
```

to

```
...
The above exception was the direct cause of the following exception:
...
```

Reply:
I never pick up on the different messages. Thanks for the explanation.